### PR TITLE
added separate map for default dimensions to add onto spans

### PR DESCRIPTION
**Pull request by harnitsignalfx**
Am I on the right track? Sorry for the crazy number of PRs, I'm closing the previous one. Ideally I'd also like to add kubernetes_node_name onto the list, but thats a separate conversation and has 2 issues that I can think of -  
a) redundant with host so not sure if that is reasonable.
b) not available in the default dimensions on the endpoint, I'd have to pass in the hostIDDims map from the writer to do it

### CODE REVIEW

File: pkg/apm/tracetracker/tracker.go [hunk 1]
```diff
@@ -23,6 +23,14 @@ var DefaultDimsToSyncSource = map[string]string{
 	"kubernetes_pod_uid": "kubernetes_pod_uid",
 }
 
+// DefaultDimsToAddToSpans are the default dimensions to add as span tags for correlated environments
+var DefaultDimsToAddToSpans = map[string]string{
```
Comment (keitwb) [inline pkg/apm/tracetracker/tracker.go:27]: Just put this in the `tracetracker` package since it isn't used at all for APM dimension update stuff.

**Review by keitwb — COMMENTED** [review]
This seems like a better solution other than the location of the map.  Fix that and I'll approve it.

**Comment by jrcamp** [comment]
lgtm, just what Ben said.

**Comment by harnitsignalfx** [comment]
Ty! Fixed, please take a look.

**Review by jrcamp — APPROVED** [review]

**Pull request merged**